docs/data.rst

```diff
@@ -13,24 +16,109 @@ Directory Setup
 Common Naming Conventions
 -------------------------
 
-`seg` is an individual river segment containing a single reach
-`hru` is a hydrologic response unit that feeds into a single seg,
+    **seg** is an individual river segment containing a single reach
+    **hru** is a hydrologic response unit that feeds into a single seg, 
     but each seg could have multiple hru's feeding into it
-`seg_id` is the identification number for a `seg`
-`site` is the gauge site name for river segments with gauge data,
-    not all segments have them
-
+    **seg_id** is the identification number for a `seg`
+    **site** is the gauge site name for river segments with gauge data, not all segments have them
 
 
 Mizuroute
 ---------
 
+``bmorph`` is designed to bias correct simulated streamflow as modeled by mizuroute_. Following
+is how to configure files for ``mizuroute`` in accoradance with the software.
+
+.. _mizuroute: https://mizuroute.readthedocs.io/en/latest/
+
+Configuration
+^^^^^^^^^^^^^
+
+Configuring files for ``mizuroute`` through ``bmorph`` should follow the following template:
+
+.. code:: ipython3
+
+    <ancil_dir>         $ancil_dir !
+    <input_dir>            $input_dir !
+    <output_dir>           $output_dir !
+    <sim_start>            $sim_start !
+    <sim_end>              $sim_end !
+    <fname_ntopOld>        $topo_file !
+    <dname_nhru>           seg !
+    <dname_sseg>           seg !
+    <seg_outlet>           -9999 !
+    <fname_qsim>           $flow_file !
+    <vname_qsim>           scbc_flow !
+    <vname_time>           time !
+    <dname_time>           time !
+    <dname_hruid>          seg !
+    <vname_hruid>          seg !
+    <units_qsim>           mm/d !
+    <dt_qsim>              86400 !
+    <is_remap>              F !
+    <restart_opt>           F !
+    <route_opt>             1 !
+    <fname_output>          $out_name !
+    <fname_state_out>       state.out.nc !
+    <param_nml>             param.nml.default !
+    <doesBasinRoute>        0 !
+    <varname_area>          Contrib_Area !
+    <varname_length>        Length !
+    <varname_slope>         Slope !
+    <varname_HRUid>         seg_id !
+    <varname_segId>         seg_id !
+    <varname_downSegId>     Tosegment !
+    <varname_hruSegId>      seg_id !
```

Comment:
We should note that the `mizuroute_utils.write_mizuroute_config` function provides a template that can be used to write a valid mizuroute configuration.